src/DaPigGuy/PiggyCustomEnchants/enchants/armor/AntiKnockbackEnchant.php

```diff
@@ -23,14 +23,6 @@ class AntiKnockbackEnchant extends ReactiveEnchantment
     /** @var int */
     public $maxLevel = 1;
 
-    /**
-     * @return array
-     */
-    public function getReagent(): array
-    {
-        return [EntityDamageByEntityEvent::class];
-    }
-
```

Comment:
This class should ``use ReactiveTrait`` instead of extending ReactiveEnchantment
> Couldn't comment line 19, so I commented this line